### PR TITLE
Fix Update Voting Method Page

### LINKED ISSUE #116: 🐛 Fix Update Voting Method Screen
**Type**: client
**FOF**:  2

## Description
- Right now, when a user click the link to update their voting method it takes them back to the original screen
- We don't want to display the initial header and also we need to display a back button

## Screenshot
Current:
image

Desired:
image

## Todo

- [ ] Fix the Update Voting Method Screen

## Links
- [Figma](https://www.figma.com/file/6aJ3cfu2Yq5dxUQcNDkm6Y/%F0%9F%8E%A8-Brink-Designs?node-id=34%3A2)

### LINKED ISSUE #118: 🗑 Clear Cache
**Type**: client
**FOF**:  1

## Description
- When a user changes their address, we want to clear their cache of favorite candidates and selected voting methods

### PR DESCRIPTION
**Pull request by ericcccsliu**
## Status:

:rocket: Ready

## Summary

Closes #116 
Closes #118 

## Changes

- Refactor voting method selection page to omit elections, include back button when reselecting voting method
- Clear context provider data (and local storage data by extension) for checkboxes and voting method selection when a new address is entered 

## Screenshots
Screen Shot 2022-04-30 at 1 16 50 AM

### CODE REVIEW

File: client/src/pages/tab/VotingMethodSelection.tsx [hunk 1]
```diff
@@ -60,34 +61,39 @@ export const VotingMethodSelection = (): ReactElement => {
...
-                <Spinner size="md" />
-              </Center>
-            )}
-          </VStack>
+          {!votingMethod && (
+            <VStack width="100%" spacing="10px">
+              {!isLoading ? (
+                <VStack width="100%" spacing="10px">
+                  <Text
+                    tabIndex={0}
```
Comment (code-arman) [inline client/src/pages/tab/VotingMethodSelection.tsx:69]: Take out tabIndex={0} for non-interactive elements

**Review by code-arman — APPROVED** [review]
LGTM

File: client/src/pages/tab/VotingMethodSelection.tsx [hunk 2]
```diff
@@ -123,11 +129,9 @@ export const VotingMethodSelection = (): ReactElement => {
               Don't see your voting method?
             </Text>
             <Text tabIndex={0}>
-              Etiam porta sem malesuada magna mollis euismod. Morbi leo risus,
-              porta ac consectetur ac, vestibulum at eros. Nullam quis risus
-              eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non
-              metus auctor fringilla. Nulla vitae elit libero, a pharetra augue.
-              Morbi leo risus, porta ac consectetur ac, vestibulum at eros.
+              Covid-19 has created some changes to how people are voting in this
```
Comment (ayanmallik) [inline client/src/pages/tab/VotingMethodSelection.tsx:132]: Thanks for updating this Covid-19 statement!

File: client/src/pages/tab/VotingMethodSelection.tsx [hunk 3]
```diff
@@ -52,6 +52,7 @@ export const VotingMethodSelection = (): ReactElement => {
   }, []);
   return (
     <main>
+      {votingMethod && <BackButton />}
```
Comment (ayanmallik) [inline client/src/pages/tab/VotingMethodSelection.tsx:55]: Great use of conditional rendering!

**Review by ayanmallik — APPROVED** [review]
Thanks for knocking out multiple issues in this PR - the updates look great so we should be good to merge this in!

**Pull request merged**